ddapm_test_agent/agent.py

```diff
@@ -402,6 +403,8 @@ async def _handle_traces(self, request: Request, version: Literal["v0.4", "v0.5"
             f.add_item(pprint.pformat(dict(headers)))
             await checks.check("meta_tracer_version_header", headers=headers)
             await checks.check("trace_content_length", headers=headers)
+            for additional_check in checks.additional:
+                await checks.check(additional_check, headers=headers)
```

Comment:
This seems brittle to me but I don't see a need to make this more robust yet since there is only a single check that relies on anything other than headers

Reply:
Although the check that @wconti27 wants to add would check the trace content so maybe I should rework it. Thoughts @Kyle-Verhoog?